### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by kvikshaug**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.travis.yml [hunk 1]
```diff
@@ -44,7 +44,11 @@ jobs:
         - docker pull {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${IMAGE_TAG}
       deploy:
         provider: script
-        script: ./scripts/push_to_hub.sh
+        script:
+         - docker tag ${IMAGE_TAG} {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${TRAVIS_COMMIT::12}
```
Comment (Midnighter) [inline {{cookiecutter.project_slug}}/.travis.yml]: I think the first part needs to be `docker tag {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${IMAGE_TAG} ...` or is this some kind of shortcut?

File: {{cookiecutter.project_slug}}/Makefile [hunk 2]
```diff
@@ -26,23 +27,23 @@ style: flake8 isort license
...
-	docker-compose run --rm -e ENVIRONMENT=testing web \
+	-docker-compose run --rm -e ENVIRONMENT=testing web \
 		/bin/sh -c "pytest -s --cov=src/{{cookiecutter.project_module}} tests"
 
 ## Run the tests and report coverage (see https://docs.codecov.io/docs/testing-with-docker).
 test-travis:
 	$(eval ci_env=$(shell bash <(curl -s https://codecov.io/env)))
 	docker-compose run --rm -e ENVIRONMENT=testing $(ci_env)  web \
-		/bin/sh -c "pytest -s --cov=src/foo_bar tests && codecov"
+		/bin/sh -c "pytest -s --cov=src/{{cookiecutter.project_module}} tests && codecov"
```
Comment (Midnighter) [inline {{cookiecutter.project_slug}}/Makefile]: Whoops, thanks :)

File: {{cookiecutter.project_slug}}/Makefile [hunk 3]
```diff
@@ -26,23 +27,23 @@ style: flake8 isort license
 
 ## Run the tests.
 test:
-	docker-compose run --rm -e ENVIRONMENT=testing web \
+	-docker-compose run --rm -e ENVIRONMENT=testing web \
```
Comment (Midnighter) [inline {{cookiecutter.project_slug}}/Makefile]: Not sure what the dash is for.

Comment (kvikshaug) [reply {{cookiecutter.project_slug}}/Makefile]: It tells Make to keep going even if the command returns with a non-zero exit code. The use case is to be able to run all QA targets locally, and not stop on the first error.

File: {{cookiecutter.project_slug}}/README.md [hunk 4]
```diff
@@ -10,6 +10,16 @@
 [![devel Codecov](https://codecov.io/gh/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/branch/devel/graph/badge.svg)](https://codecov.io/gh/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/branch/devel)
 [![devel Requirements Status](https://requires.io/github/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/requirements.svg?branch=devel)](https://requires.io/github/{{cookiecutter.github_username}}/{{cookiecutter.project_slug}}/requirements/?branch=devel)
 
+## Post-cookiecutter steps
+
+Perform the following steps after creating a new service from the cookiecutter.
+
```
Comment (Midnighter) [inline {{cookiecutter.project_slug}}/README.md]: Also set the `${DOCKER_PASSWORD}`.

File: {{cookiecutter.project_slug}}/.travis.yml [hunk 5]
```diff
@@ -44,7 +44,11 @@ jobs:
         - docker pull {{cookiecutter.docker_username}}/{{cookiecutter.project_slug}}:${IMAGE_TAG}
       deploy:
         provider: script
-        script: ./scripts/push_to_hub.sh
+        script:
```
Comment (Midnighter) [inline {{cookiecutter.project_slug}}/.travis.yml]: Just a note: Travis will run all the commands here even if one of them errors. Of course, these should not be causing an error :)

File: {{cookiecutter.project_slug}}/README.md [hunk 6]
```diff
@@ -14,10 +14,12 @@
 
 Perform the following steps after creating a new service from the cookiecutter.
 
-* Create the following environment variables in Travis CI for Slack notifications:
+* Create the following environment variables in Travis CI:
   * `SLACK_ACCOUNT`: Workspace name, e.g. `biosustain`
   * `SLACK_TOKEN`: [Find it here](https://biosustain.slack.com/services/B8D8VKW3W)
   * `SLACK_CHANNEL`: Normally `#decaf-notifications`
+  * `DOCKER_PASSWORD`: For push access to [Docker Hub](https://hub.docker.com/u/dddecaf/dashboard/)
+  * `CODECOV_TOKEN`: For push access to [codecov.io](https://codecov.io/gh/DD-DeCaF)
```
[the remark this answers is not in the document]
Comment (kvikshaug) [reply {{cookiecutter.project_slug}}/README.md]: Ah, you're right. I'll remove it from the model service settings.

File: {{cookiecutter.project_slug}}/.travis.yml [hunk 7]
```diff
@@ -20,11 +20,11 @@ matrix:
   fast_finish: true
 
 before_script:
-  - docker login -u={{cookiecutter.docker_username}} -p=${DOCKER_PASSWORD}
+  - docker login -u=${DOCKER_USERNAME} -p=${DOCKER_PASSWORD}
```
Comment (Midnighter) [inline {{cookiecutter.project_slug}}/.travis.yml]: We could also hard code this to `decaftravis` in order to create less incentives to put in personal credentials?

**Pull request merged**